src/lib/support/jsontlv/TextFormat.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2013-2017 Nest Labs, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <algorithm>
+#include <json/json.h>
+
+namespace chip {
+std::string PrettyPrintJsonString(const std::string & jsonString)
+{
+    Json::Reader reader;
+    Json::Value jsonObject;
+    reader.parse(jsonString, jsonObject);
+    Json::StyledWriter writer;
+    return writer.write(jsonObject);
+}
+
+std::string MakeJsonSingleLine(const std::string & jsonString)
```

Comment:
Concretely: how about we just remove MakeJsonSingleLine?  This implementation of it is broken and can't be used for anything serious; if we ever need something like this, we will need to write an implementation that actually works.